pandas/io/formats/format.py

```diff
@@ -127,6 +127,11 @@
             name.
             The result of each function must be a unicode string.
             List/tuple must be of length equal to the number of columns.
+
+                NOTE: formatters are **not** applied to missing values.
+                Use the ``na_rep`` parameter instead, or see:
+                https://pandas.pydata.org/docs/dev/user_guide/io.html#writing-a-formatted-string
```

Comment:
I'd prefer if this were inline with the rest of the description above. For the link, it should be a [cross-reference](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html) rather than an external link.

Reply:
Whoops, mybad. I completely agree, thanks for the reference.

I can't quite tell if cross-references are working correctly in text lifted from `.py` files (the containerised build process is good, but does throw some odd errors and is quite slow for a full build). I'll check the previewer action.